Source/Client/Effects/Lightning.cs

```diff
@@ -69,38 +70,46 @@ public Lightning(ILightningNode source, float sourceheight, ILightningNode targe
 				// Set up heavy settings
 				fadechange = FADE_CHANGE_BIG;
 
-				// Play blast sound
-				if(blastsound) DirectSound.PlaySound(SND_FILE_START, MakeMiddlePosition());
-			}
+
+                string snd = DirectSound.GetSound("SND_FILE_START", false);
```

Comment:
Are you sure this should be in quotes? `"SND_FILE_START"`, not `SND_FILE_START`?

Reply:
Yes you right)